client/my-sites/plan-features-comparison/header.jsx

```diff
@@ -79,6 +79,8 @@ export class PlanFeaturesComparisonHeader extends Component {
 		const { isMonthlyPlan, rawPriceForMonthlyPlan, annualPricePerMonth, translate } = this.props;
 
 		if ( ! isMonthlyPlan ) {
+			const isLoading = ! rawPriceForMonthlyPlan;
```

Comment:
```suggestion
			const isLoading = typeof rawPriceForMonthlyPlan !== 'number';
```

Would something like this be more appropriate? I know the free plan is currently excluded from the comparison table so `rawPriceForMonthlyPlan` won't be `0`. But in the interest of making the comparison resilient to future changes.